### PR TITLE
DEV-8202 Npm attempt

### PR DESCRIPTION
**Pull request by nick-torres**
**High level description:**

node, npm, and travis build update

**JIRA Ticket:**
[DEV-8202](https://federal-spending-transparency.atlassian.net/browse/DEV-8202)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [ ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,17 +22,17 @@ _Get the code and install the runtime and dependencies_
...
+1. Set Node Package Manager (`npm`) CLI to version `8.5.0`.
     ```shell
-    $ npm i -g npm@6.14.13
+    $ npm i -g npm@8.5.0
     ```
-1. Perform a clean install to get an exact dependency tree:
+1. Perform an install to get an exact dependency tree:
     ```shell
-    $ npm ci
+    $ npm install --legacy-peer-deps
```
Comment (anjenkin) [inline README.md:35]: We should stick with npm ci here so the command is `npm ci --legacy-peer-deps` to make sure folks are doing a clean install

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,9 @@ cache:
   directories:
     - node_modules
 before_install:
-  - npm install -g @lhci/cli@0.4.x
+  - npm install -g @lhci/cli@0.9.x
+install:
+  - npm install --legacy-peer-deps
```
Comment (anjenkin) [inline .travis.yml:20]: should this use `npm ci` instead of npm install?

**Review by anjenkin — CHANGES_REQUESTED** [review]
just a couple comments.  also the node version in the .nvmrc file should be changed

File: .travis.yml (hunk 2 above)
Comment (nick-torres) [reply .travis.yml:20]: i don't know if travis actually caches anything so im not sure there is a difference between ci and install when it comes to travis

File: README.md (hunk 1 above)
Comment (nick-torres) [reply README.md:35]: good call, fixing

File: .travis.yml (hunk 2 above)
Comment (nick-torres) [reply .travis.yml]: updated just in case

**Comment by nick-torres** [comment]
> just a couple comments. also the node version in the .nvmrc file should be changed

changed nvmrc as well, good catch

**Pull request merged**